packages/elements/src/interactive-chart/index.ts

```diff
@@ -506,8 +488,7 @@ export class InteractiveChart extends ResponsiveElement {
 
       for (let index = 0; index < this.internalConfig.series.length; index++) {
         // Get seriesOptions and type
-        const seriesOptions =
-          (this.internalConfig.series[index].seriesOptions as ChartSeriesOptions<SeriesStyleOptions>) || {};
+        const seriesOptions = (this.internalConfig.series[index].seriesOptions as SeriesOptions) || {};
```

Comment:
Partial would be better here.
```suggestion
        const seriesOptions: Partial<SeriesOptions> = this.internalConfig.series[index].seriesOptions || {};
```